### DISCUSSION TITLE
Advanced button in Security tab stopped being visible

### DISCUSSION THREAD

**Discussion by KodeXall** [keepassxreboot/keepassxc#9286] [Q&A] [answered]
Some time ago I was adjusting the _decryption time_ option in my database (https://github.com/keepassxreboot/keepassxc/discussions/9183) - and I did do what I wanted to. But now it seems that in the left bottom corner I no longer have the _**Advanced**_ button at all: i.e. it is not merely grayed out when not being in the **_Security_** tab, but it is completely absent

I would post a screenshot, but apparently window of **_KeePassXC 2.7.4_** is transparent to the action of pressing of _**Print Screen**_ key executed on **_Windows 10 Enterprise 20H2 19042 x64_**

**Comment by droidmonkey** [comment]
![image](https://user-images.githubusercontent.com/2809491/229916048-5437d333-4388-4142-93ad-1c1e605dea9d.png)

seems to be fine for me.

**Comment by KodeXall** [comment]
My sidebar looks different

How do I take a screenshot of it? Its "invisible" to my **_Prnt Scr_** key on Windows 10

**Reply by droidmonkey** [reply]
You need to stay the program with `--allow-screencapture`

**Comment by KodeXall** [comment, marked as answer]
Here is what I get: to see in my _**2.7.4**_:

![KeePassXC settings](https://user-images.githubusercontent.com/107837309/230078631-82606bc2-5e3c-49a5-a3fe-07b01bcf13b1.jpg)

**Reply by droidmonkey** [reply]
You are in application settings, not database settings

**Comment by KodeXall** [comment]
I must have suffered a temporal mental lapse. I am sorry for wasting your time

And thus to minimize possibility of occurrence of such topics as this one, coming out from other new users, I have made a new proposition: https://github.com/keepassxreboot/keepassxc/discussions/9300
